thoth/common/openshift.py

```diff
@@ -1461,12 +1461,26 @@ def get_build_dependencies_template(self) -> Dict[str, Any]:
         return self._get_template("template=build-dependencies")
 
     @staticmethod
-    def _verify_thoth_integration(source_type: str):
+    def _verify_thoth_integration(source_type: Optional[str]):
         """Verify if Thoth integration exists."""
-        integrations = [i.name for i in ThothAdviserIntegrationEnum]
+        if source_type not in ThothAdviserIntegrationEnum.__members__ and source_type is not None:
```

Comment:
this should be `.__members__.keys()`

Reply:
Oh wait, I forgot about behavior of in with dictionaries, this is actually good

Reply:
The dict read only the keys if you don't state differently :) 

for `k in dict_name` or  `for k in dict_name.keys()` should be equivalent